00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,593 @@
+
+Once this has been repeated for all tasks in Accepted, the workload for the
+coming sprint has been defined, and sprint planning is complete.
+
+This meeting should be scheduled for the hour immediately following the
+review. The two combined meetings should collectively replace the daily standup
+on the first day of a new sprint. Here's a possible time breakdown:
+
+- 10 min: triage
+- 5 min: last-minute issue nomination
+- 30 min: discussion of all nominated issues and whether they are project
+  priorities 
+- 15 min: volunteering and assignment
+
+### Daily Standup
+
+Daily standup is the team's daily check-in meeting, and when the **Triage**
+process is performed. It should provide an opportunity for individual
+contributors (and guests) to ask questions, elaborate on blockers, and ask for
+help if needed.
+
+As noted before, triage is the single most important component of this meeting,
+and should therefore be done first. The exact process of triage is detailed
+separately in the [Triage](#triage) section. Triage should not be strictly
+time-bound; if productive discussions are happening around a new issue or PR,
+those should be allowed to happen naturally.
+
+The decision to make triage a part of daily standup is deliberate: it allows
+issues and PRs filed by "drive-by" contributors (that is, people not part of
+the project that still wish to contribute) to receive human consideration after
+a maximum of 24 working hours -- a desirable property for open source.
+
+Following triage, the scrum master will pull up the sprint board for an informal
+board review. This does not need to go over every single issue; rather, it
+should be an opportunity for team members to point out highlights or interesting
+updates to their tasks, and request help on blockers or otherwise. To facilitate
+this, the scrum master should ask the meeting's attendees for a quick show of
+hands on if they have something they wish to talk about; this will allow the
+scrum master to keep time constraints in mind and move conversations along if
+they're taking up too much time.
+
+Finally, the rest of the meeting should be dedicated to open-ended project
+discussion. If there are guests to the call, the team should welcome them and
+allow them to ask questions. If there are any other topics to cover, they should
+be brought up here.
+
+Standup should take no longer than a half-hour, and should be scheduled as
+follows:
+
+- 10-15 min: triage
+- 15-20 min: Board review/blockers/project discussion (however long is left
+  after triage)
+
+### Review
+
+The final meeting is **Review**, which represents the end of a
+sprint. It is a time for the whole team to reflect on work accomplished, demo
+new things, and discuss how to approach tasks that didn't get completed.
+
+Most of these topics will take the form of discussion. The only significant
+process to detail here is **Retrospective**, the workflow for reviewing issues
+and PRs from the last sprint.
```

Comment:
I think calling this meeting **Review** is a good idea, however i take issue with calling **Retrospective** something that is disconnected from the [accepted meaning](http://retrospectivewiki.org/index.php?title=Agile_Retrospective_Resource_Wiki) of an agile retrospective, especially if we are telling folks we are basing our process on agile processes.

In light of this,
1. we should actually make time for engaging in retrospectives, as they are a good tool to ensure the "agile scaffolding" so to speak is working out for the team members and
2. call what is currently being termed "retrospective" an issue review, a card review, an end of sprint triage, or something like that.

Reply:
While perhaps not explicitly stated, I intended for point 1) to be addressed in the last 20 minutes of the meeting ("demos or other discussion" -- which should be simplified to "other discussion".)

I will clarify that those last 20 minutes should be for "retrospective".

As for 2)... I'm not really a fan of conflating "triage" with an "end of sprint triage". What about something like "end of sprint issue cleanup"?